### ISSUE TITLE
Add a health status metric for every gpu card

### ISSUE THREAD

**Issue by lx1036** [NVIDIA/dcgm-exporter#384]
### Ask your question

I'm curious, why aren't there any health status metrics for every GPU card?

I check the NVIDIA/go-dcgm has function like HealthCheckByGpuId(gpuId uint) https://github.com/NVIDIA/go-dcgm/blob/main/pkg/dcgm/api.go#L102-L105 , and if we can get health status for every card, and export health metric.

**Comment by glowkey** [comment]
The next major version of DCGM-Exporter planned for sometime in the next few months will have a capability like this. Stay tuned.

**Comment by wumuran96** [comment]
Has the new capability mentioned for the next major version of DCGM-Exporter been implemented yet?

**Comment by glowkey** [comment]
Yes, you can now add `DCGM_EXP_GPU_HEALTH_STATUS, gauge, GPU health status` to the list of watched metrics.

**Comment by nccurry** [comment]
Add `DCGM_EXP_GPU_HEALTH_STATUS` to your custom metrics CSV. Closing.
